plutus-ledger/src/Ledger/Scripts.hs

```diff
@@ -1,7 +1,8 @@
-module Ledger.Scripts (
-    module Export
-    ) where
+module Ledger.Scripts
+  ( module Plutus.V1.Ledger.Scripts
+  , module Plutus.Script.Utils.V1.Scripts
```

Comment:
Isn't it simpler to do `module Export` here?

Reply:
It might be a personal preference but I find hard to figure out what I am exporting with the `as Export` pattern. In particular I had to fix a bunch of conflicting exports (some modules would rexport `Plutus.V1.Ledger.Tx.TxOut` instead of `Plutus.Ledger.Tx.TxOut`) and hls was not highlighting correctly which module was causing the problem (it was only mentioning `Export`).